Release Note for WebView2 with Stable 101 and prerelease 102 (#1924)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/release-notes.md

To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.

To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).

<!-- ====================================================================== -->

## 1.0.1210.30
  
Release Date: May 2, 2022  
  
[NuGet package for WebView2 SDK 1.0.1210.30](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1210.30)  
  

> For full API compatibility, 101.1210.30 or higher.  

Reviewer: ```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 101.1210.30 or higher.  

```

### General

#### Promotions

The following items are now stable:

*   Support for [multiple user profiles](/microsoft-edge/webview2/reference/win32/icorewebview2environment10?view=webview2-1.0.1210.30) in WebView2.

*	[Theming API](/microsoft-edge/webview2/reference/win32/icorewebview2profile?view=webview2-1.0.1210.30) which provides a way to customize the WebView2 color theme as `light`, `dark`, or `system`.

*	[Default Download API](/microsoft-edge/webview2/reference/win32/icorewebview2profile?view=webview2-1.0.1210.30) which provides a way to customize the default download location.

<!-- ====================================================================== -->
## 1.0.1243-prerelease
  
Release Date: May 2, 2022  
  

> [NuGet package for WebView2 SDK 1.0.1243](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1243-prerelease)  

Reviewer: ```suggestion
[NuGet package for WebView2 SDK 1.0.1243-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1243-prerelease)  

```

> For full API compatibility, 102.0.1243.0 or higher.  

Reviewer: ```suggestion
For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 102.0.1243.0 or higher.  

```

### General

#### Promotions

> *	Add [ServerCertificateErrorDetected API](/microsoft-edge/webview2/reference/win32/icorewebview2_13?view=webview2-1.0.1243-prerelease) which allows you to hanle the event when the WebView2 cannot verify server's digital certificate.

Reviewer: ```suggestion
*	Add [ServerCertificateErrorDetected API](/microsoft-edge/webview2/reference/win32/icorewebview2_13?view=webview2-1.0.1243-prerelease) which allows you to handle the event when the WebView2 control cannot verify server's digital certificate.
```

> * 	The [ClearBrowsingData API](/microsoft-edge/webview2/reference/win32/icorewebview2profile2?view=webview2-1.0.1243-prerelease):

Reviewer: do like the other ?view instances; the doc build will tell you to add the &preserve.... portion of URL.

> 	*  clearBrowsingDataInTimeRange
> 	*  clearBrowsingDataAll

Reviewer: ```suggestion
	*  `clearBrowsingDataInTimeRange`
	*  `clearBrowsingDataAll`
```
if I understand, the plan is to link all these
